keyboards/packrat/readme.md

```diff
@@ -0,0 +1,17 @@
+# Packrat
+
+![Packrat](https://i.imgur.com/t6vWYIb.jpeg)
+
+Full Gallery: https://imgur.com/gallery/fZklCG6
+
+Packrat is an 11u wide ortholinear keyboard featuring multiple bottom row options and top-centered rotary encoder support.
+
+* Keyboard Maintainer: [H. Bond](https://github.com/drhigsby)
+* Hardware Supported: Packrat (https://github.com/drhigsby/packrat)
+* Hardware Availability: cases available at TBD / make your own
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

Reply:
Yes, I will address this.

Reply:
Addressed in this commit https://github.com/qmk/qmk_firmware/pull/14574/commits/a6ef25ac8e4b09f26cfda50c8a0418056f69f313